src/components/Stepper/Stepper.tsx

```diff
@@ -359,28 +466,92 @@ export const Stepper: FC<StepperProps> = React.forwardRef(
                 >
                   <ul className={styles.steps} ref={stepsRef}>
                     {steps.map((step: Step, index: number) => {
-                      let stepItem = null;
+                      const combinedStepSize: StepSize = getCombinedStepSize(
```

Comment:
what is a combinedStep? bit confused about this. Might be worth some code documentation

Reply:
Each step has a unique set of props, this takes the aggregate and determines the overall spacing of the timeline.